sorc/ncep_post.fd/xml_perl_data.f

```diff
@@ -20,14 +35,22 @@ module xml_perl_data
 	    character(len=10)                    :: table_info=''
 	    character(len=80)                    :: stats_proc=''
 	    character(len=80)                    :: fixed_sfc1_type=''
-            integer, dimension(:), pointer       :: scale_fact_fixed_sfc1 => null()
-	    real, dimension(:), pointer          :: level => null()
-	    character(len=80)                    :: fixed_sfc2_type=''
-	    integer, dimension(:), pointer       :: scale_fact_fixed_sfc2 => null()
-	    real, dimension(:), pointer          :: level2 => null()
-	    character(len=80)                    :: aerosol_type=''
+       integer, dimension(:), pointer       :: scale_fact_fixed_sfc1 => null()
+	    
```

Comment:
@gspetro-NOAA Is there a special reason for adding a blank line here and the next several lines?

Reply:
@WenMeng-NOAA I was originally trying solutions for documenting variables with a "pointer" label because Doxygen doesn't seem to parse those variables properly, and the blank lines and spacing changes got in that way. Ultimately, I just had to do a general group definition for the variables. 
This is one file that did not have a warning attached to it in Issue #392, but when I resolved an error message related to it, Doxygen suddenly started asking for variable definitions. It would be good to come back to it at a future date to provide additional information for users.

Reply:
@gspetro-NOAA Got it. Thanks for explaining.